### PR TITLE
Direct series link

### LINKED ISSUE #359: Add some form of direct link route with series UUID
A common requirement by external tools is having a fixed URL for Tobira that just includes a series' UUID and renders some information about the series. We need to add such a route. We likely also want to add the same with the Tobira ID instead of the series UUID. 

It's still unclear what the route would do exactly:
- Either show information about the series directly
- Or forward to another route where the series is mounted on

### PR DESCRIPTION
**Pull request by JulianKniephoff**
Closes #359.

Can be reviewed commit by commit.

Notably missing is any sort of check whether a user "is allowed" to view a given series, because we haven't figured out what that even means. See #398 .

### CODE REVIEW

**Review by LukasKalbertodt — CHANGES_REQUESTED** [review]
Generally looks good, but I left some remarks.

Also: Please add a DB index for Opencast-ID lookups.

File: frontend/src/routes/Video.tsx [hunk 1]
```diff
@@ -47,42 +47,44 @@ export const VideoRoute = makeRoute(url => {
...
+            )
+            : ({ event, realm }) => (
+                !event || !realm || !realm.referencesVideo
+                    ? <NotFound kind="video" />
+                    : <VideoPage
+                        {...{ event, realm, id }}
+                        basePath={realmPath.replace(/\/$/u, "") + "/v"}
+                    />
+            );
+
```
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply frontend/src/routes/Video.tsx:82]: For the record: I disagree that having the start of the condition on the same line as a lengthy type signature is equally as readable, and even with the additional indentation, the entire thing is also still comfortably under 80 chars. :shrug: Indentation in itself is not evil.

Anyway, your opinion seemed to be harder than mine, so whatever.

File: frontend/src/routes/Video.tsx [hunk 2]
```diff
@@ -54,35 +54,37 @@ export const DirectVideoRoute = makeRoute(url => {
...
+    const isDirectLink = realmPath === undefined;
     const queryRef = loadQuery<VideoQuery>(query, { id, realmPath: realmPath ?? "/" });
 
+    const render: (result: VideoQuery$data) => JSX.Element
+        = isDirectLink
+            ? ({ event, realm }) => (
+                !event
+                    ? <NotFound kind="video" />
+                    : <VideoPage
+                        {...{ event, realm: realm ?? unreachable("root realm doesn't exist"), id }}
```
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply frontend/src/routes/Video.tsx:70]: I actually shuffled some more things around. See the new commit(s) and let me know what you think. :wink:

File: frontend/src/routes/Series.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { Nav } from "../layout/Navigation";
+import { PageTitle } from "../layout/header/ui";
+
+import { NotFound } from "./NotFound";
+import { SeriesByOpencastIdQuery } from "./__generated__/SeriesByOpencastIdQuery.graphql";
+
+
+export const DirectSeriesRoute = makeRoute(url => {
+
+    const regex = new RegExp("^/!s/:(.+)", "u");
```
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply frontend/src/routes/Series.tsx:20]: You mean `/!s/:foo/bar/banana`, right? :thinking: I mean yes, probably not important, but I can probably also easily fix it. :shrug:

File: frontend/src/routes/Series.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                if (!series) {
+                    return <NotFound kind="series" />;
+                }
+
+                const { t } = useTranslation();
+
+                return <>
+                    <PageTitle title={series.title} />
+                    <p>{series.description}</p>
+                    <SeriesBlockFromSeries
```
[the remark this answers is not in the document]
Comment (JulianKniephoff) [reply frontend/src/routes/Series.tsx]: Probably even if there is a description. :shrug:

**Review by LukasKalbertodt — APPROVED** [review]

**Pull request merged**